src/Modules/ConfigOptions.lua

```diff
@@ -1718,21 +1715,45 @@ Huge sets the radius to 11.
 	
 	-- Section: Custom mods
 	{ section = "Custom Modifiers", col = 1 },
-	{ var = "customMods", type = "text", label = "", apply = function(val, modList, enemyModList)
-		for line in val:gmatch("([^\n]*)\n?") do
-			local mods, extra = modLib.parseMod(line)
+	{ var = "customMods", type = "text", label = "",
+		apply = function(val, modList, enemyModList, build)
+			for line in val:gmatch("([^\n]*)\n?") do
+				local strippedLine = StripEscapes(line):gsub("^[%s?]+", ""):gsub("[%s?]+$", "")
+				local mods, extra = modLib.parseMod(strippedLine)
 
-			if mods then
-				local source = "Custom"
-				for i = 1, #mods do
-					local mod = mods[i]
+				if mods then
+					local source = "Custom"
+					for i = 1, #mods do
+						local mod = mods[i]
 
-					if mod then
-						mod = modLib.setSource(mod, source)
-						modList:AddMod(mod)
+						if mod then
+							mod = modLib.setSource(mod, source)
+							modList:AddMod(mod)
+						end
```

Comment:
Well inactiveText is called every frame right so i assume modifying modList every frame would have significant impact to performance. And the parsing logic should never be called twice as the apply is being called when the textx is changing and the inactiveText is being called when out of focus, 2 events that do not happen at once. There is maybe something to be said about apply only being called after you stop typing (e.g adding some typing buffer/delay) but I would probs want to handle that in separate PR as I would like to do that for other types of controls too (like counters) and the issue also happens on master.

Reply:
Hmm, yeah I didn't realize the `inactiveText` was called each frame.  Luckily we cache the mods, otherwise even that parse could get expensive.  By happening twice I meant happens when you type the mod, then happens again when you unfocus the box.

There is an OnFocusLost method you could implement, but you'd have to store the inactiveText somehow so the edit control can use it permanently.

Reply:
I originally actually stored inactiveText but the issue is that now its dynamic with addition of ALT pressing clearing the recolors. Originally it was stored in the apply method and then simply displayed.